tests/test_action.py

```diff
@@ -19,6 +19,56 @@
 from lux.vis.Vis import Vis
 
 
+def test_temporal_action(global_var):
+    airbnb_df = pd.read_csv(
+        "https://raw.githubusercontent.com/lux-org/lux-datasets/master/data/airbnb_nyc.csv"
+    )
+    flights_df = pd.read_csv(
+        "https://raw.githubusercontent.com/lux-org/lux-datasets/master/data/flights.csv"
+    )
+    date_df = pd.DataFrame(
+        {
+            "date": [
+                "2019-01",
+                "2014-02",
+                "2020-03",
+                "2013-04",
+                "2012-05",
+                "2019-01",
+                "2020-02",
+                "2013-03",
+                "2020-04",
+                "2000-05",
+                "2000-01",
+                "2000-02",
+                "2004-12",
+                "2004-06",
+                "2020-05",
+                "2020-01",
+                "2020-02",
+                "2020-03",
+                "2020-04",
+                "2020-05",
+            ]
+        }
+    )
+    test_data = [airbnb_df, flights_df, date_df, pytest.car_df, pytest.olympic]
```

Comment:
For the flights dataset, the results look a bit weird since the different timescales are created even for attributes that only encompass a single timescale.
```python
df = pd.read_csv("../../lux-datasets/data/flights.csv")
df['year'] = pd.to_datetime(df['year'], format='%Y')
df['month'] = pd.to_datetime(df['month'], format='%M')
df['day'] = pd.to_datetime(df['day'], format='%d')
```
![image](https://user-images.githubusercontent.com/5554675/107752470-bcae7980-6d59-11eb-962c-375582b5dde3.png)

Reply:
I'll take a closer look at these results. The fixes/changes to utilize the datetime attribute and discarding timescales that are not applicable should end up resolving this issue. I'll make the changes and ensure that these visualizations are proper.

Reply:
I noticed in your updated [results](https://github.com/lux-org/lux/pull/262#issuecomment-787717020) that the range for the y-axis doesn't fit the data well (e.g. the minimum of the data points are far higher than the minimum of the y-range). Do you know why that might be?